### PR TITLE
add max width for images inside panel-body

### LINKED ISSUE #242: Images come outside the thread/reply borders
Example: https://laravel.io/forum/gameq-v3-integration-to-laravel-54

Probably just needs a max-width for images.

screen shot 2017-06-19 at 2 45 36 pm

### PR DESCRIPTION
**Pull request by fetzi**
add max-width for all images inside `panel-body`, fixes #242

### CODE REVIEW

File: resources/assets/sass/_forum.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.panel-body {
+  img {
+    max-width: 100%;
+  }
+}
```
Comment (driesvints) [inline resources/assets/sass/_forum.scss]: Heya! Can you use 4 spaces for indentation? Thanks :)

**Comment by driesvints** [comment]
Thanks for sending this in! I'll merge it if you fix the indentation.

**Comment by fetzi** [comment]
@driesvints fixed!

File: resources/assets/sass/app.scss [hunk 2]
```diff
@@ -12,3 +12,4 @@
 @import "nav";
 @import "profile";
 @import "wysiwyg";
+@import "forum";
```
Comment (driesvints) [inline resources/assets/sass/app.scss]: Ah sorry, I missed this earlier. Can you place forum right underneath "footer" so they're ranked alphabetically?

**Comment by fetzi** [comment]
done

**Comment by driesvints** [comment]
Thanks!

**Comment by driesvints** [comment]
@fetzi hmm unfortunately this broke user images on threads on the forum overview: https://laravel.io/forum

Can you perhaps send in a follow-up pr so the css rule you added specifically targets thread and reply bodies?

**Pull request merged**